charts/helm-chart/kubernetes-dashboard/values.yaml

```diff
@@ -85,14 +85,15 @@ app:
     # https://localhost:8443
     - localhost
     # - kubernetes.dashboard.domain.com
-    ingressClassName: nginx
+    ingressClassName: internal-nginx
```

Comment:
@floreks - Private ingress class name can be anything for the users. e.g. "nginx-private" or "private-nginx" or "nginx-pvt" etc. How would the helm chart behave in such scenarios?

Reply:
This would mean that you already have nginx installed and you should not use our defaults, but rather override this with your custom value.